ci-operator/config/shiftstack/shiftstack-ci/shiftstack-shiftstack-ci-main__capi-periodic-4.16.yaml

```diff
@@ -0,0 +1,255 @@
+base_images:
+  manila-csi-driver-operator-test:
+    name: "4.16"
+    namespace: ocp
+    tag: manila-csi-driver-operator-test
+  openstack-cinder-csi-driver-operator-test:
+    name: "4.16"
+    namespace: ocp
+    tag: openstack-cinder-csi-driver-operator-test
+  openstack-installer:
+    name: "4.16"
+    namespace: ocp
+    tag: openstack-installer
+  openstack-tests:
+    name: "4.16"
+    namespace: ocp
+    tag: openstack-tests
+  stolostron:
+    name: builder
+    namespace: stolostron
+    tag: go1.21-linux
+releases:
+  initial:
+    candidate:
+      product: ocp
+      relative: 1
+      stream: ci
+      version: "4.16"
+  latest:
+    candidate:
+      product: ocp
+      stream: ci
+      version: "4.16"
+resources:
+  '*':
+    limits:
+      memory: 4Gi
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tests:
+- as: e2e-openstack-fips
+  minimum_interval: 56h
```

Comment:
I see the new jobs have the same `minimum_interval` as the periodics without CAPI.
Is there a chance jobs with the same `minimum_interval` will run at the same time? we might see etcd issues if that is true. I suggest tweaking this a bit.

Reply:
We designed minimum_interval to address this problem exactly: each job's interval will start after it's completed, rather than after it started. As a consequence, jobs delayed because they couldn't immediately acquire a lease are naturally moved forward until they find a stable spot.